services/node_create.proto

```diff
@@ -0,0 +1,86 @@
+syntax = "proto3";
+
+package proto;
+
+/*
+ * Hedera Network Services Protobuf
+ *
+ * Copyright (C) 2024 Hedera Hashgraph, LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+option java_package = "com.hederahashgraph.api.proto.java";
+// <<<pbj.java_package = "com.hedera.hapi.node.dynamic">>> This comment is special code for setting PBJ Compiler java package
+option java_multiple_files = true;
+
+import "basic_types.proto";
+
+/**
+ * A transaction body to add a new node to the network.
+ *
+ * This transaction body SHALL be considered a "privileged transaction".
+ *
+ * This message supports a transaction to create a new node in the network.
+ * The transaction, once complete, enables a new consensus node
+ * to join the network, and requires governing council authorization.
+ *
+ * A `NodeCreateTransactionBody` MUST be signed by the governing council.<br/>
+ * The newly created node information will be used to generate config.txt and public.pfx during phase 2,<br>
+ * not active until next freeze upgrade.
+ */
+message NodeCreateTransactionBody {
```

Comment:
We should list the effect on the TransactionRecord for each transaction type. That is, here we should list that the newly allocated node ID will be populated in the record.